File: SmokeSaver/Models/CatalogProduct.cs
namespace SmokeSaver.Models;

public sealed class CatalogProduct
{
    public string Id { get; set; } = string.Empty;

    public string Title { get; set; } = string.Empty;

    public decimal PriceUah { get; set; }

    public string Category { get; set; } = string.Empty;

    public string ShortDescription { get; set; } = string.Empty;
}

File: NitKotin.Android/app/src/main/java/com/nitkotin/android/data/model/CatalogProduct.kt
package com.nitkotin.android.data.model

import kotlinx.serialization.Serializable
import kotlinx.serialization.SerialName

@Serializable
data class CatalogProduct(
    @SerialName("Id")
    val id: String,
    @SerialName("Title")
    val title: String,
    @SerialName("PriceUah")
    val priceUah: Double,
    @SerialName("Category")
    val category: String,
    @SerialName("ShortDescription")
    val shortDescription: String,
)
